### PR TITLE
challenge 18

### PR DESCRIPTION
**Pull request by bashamtg**
- [X] Top-level README “Table of Contents” is updated
 - [x] README for this challenge is complete
   - [x] Summary, Description, Approach & Efficiency, Solution
   - [x] Picture of whiteboard

 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
   - [x] “Happy Path” - Expected outcome
   - [x] Expected failure
   - [x] Edge Case (if applicable/obvious)

### CODE REVIEW

File: python/code_challenges/tree_breadth_first.py [hunk 1]
```diff
@@ -42,3 +42,25 @@ def breadth_first(tree):
...
 
     return values
+
+
+
+def fizz_buzz_tree(k_ary_tree):
+    breadth_first_order = breadth_first(k_ary_tree)
+    result = []
+    for node in breadth_first_order:
+        if node % 3 == 0 and node % 5 == 0:
```
Comment (aaron-imbrock) [inline python/code_challenges/tree_breadth_first.py:52]: I'm sure this works fine but it would be better to write a separate function that implements the code here and then run it against node.value.
